packages/compass-components/src/components/keyline-card.tsx

```diff
@@ -17,23 +20,74 @@ const keylineDarkThemeStyles = css({
   borderColor: palette.gray.dark2,
 });
 
+const lightHoverBoxShadow = `0 4px 20px -4px ${transparentize(
+  0.8,
+  palette.black
+)}`;
+
+const darkHoverBoxShadow = `0 4px 20px -4px ${transparentize(
+  0.3,
+  palette.black
+)}`;
+
+const clickableStyleLight = css`
+  cursor: pointer;
+
+  &:hover {
+    border: 1px solid ${palette.gray.light2};
+    box-shadow: ${lightHoverBoxShadow};
+  }
+
+  &:active {
+    box-shadow: none;
+  }
+`;
+
+const clickableStyleDark = css`
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: ${darkHoverBoxShadow};
+  }
+`;
```

Comment:
To make the hover smoother, I'm thinking lets pull over the transition styles from lg:
https://github.com/mongodb/leafygreen-ui/blob/cf06988dcef8e2dac83ebc409f3b2e9a905fe9e8/packages/card/src/Card/styles.ts#L86C1-L87C43

```
import {
  transitionDuration
} from '@leafygreen-ui/tokens';
```


```suggestion
const clickableStyleLight = css`
  cursor: pointer;
  transition: ${transitionDuration.default}ms ease-in-out;
  transition-property: border, box-shadow;

  &:hover {
    border: 1px solid ${palette.gray.light2};
    box-shadow: ${lightHoverBoxShadow};
  }

  &:active {
    box-shadow: none;
  }
`;

const clickableStyleDark = css`
  cursor: pointer;
  transition: ${transitionDuration.default}ms ease-in-out;
  transition-property: border, box-shadow;

  &:hover {
    box-shadow: ${darkHoverBoxShadow};
  }
`;
```

A total nit, but we might want to standardize on using `css({...})` vs 'css`...`' in these places. Although this is pulled from LG that uses 'css`...`'.

Reply:
Sounds great!

Reply:
done